src/TestRunnerOrchestrator.ts

```diff
@@ -168,7 +168,7 @@ export default class TestRunnerOrchestrator {
   }
 
   private createTestRunnerSandboxes(): Promise<TestRunnerSandbox[]> {
-    let cpuCount = os.cpus().length;
+    let cpuCount = this.options.cpuCount || os.cpus().length;
```

Comment:
We use the cpuCount to determine how many test runners should be used in parallel. I think it's a good idea to name the option after this purpose. Something like `testRunnerCount`. What do you think?